### DISCUSSION TITLE
interpolation

### DISCUSSION THREAD

**Discussion by esquilo444** [datarhei/restreamer#682] [Q&A]
Good morning,

Small subsidiary question.
I regularly use interpolation to adapt the frame rate as closely as possible depending on the destination server in order to have the most comfortable rendering for the viewer.
I would like to know which ffmpeg interpolation mode you are using. BLEND OR MOTION ESTIMATION?
Thank you in advance for your light

Beautiful day

Jerome

**Comment by ioppermann** [comment]
We are using the `framerate` filter (https://ffmpeg.org/ffmpeg-filters.html#framerate) from ffmpeg with the desired target framerate. All other options are left to their default value.

**Comment by esquilo444** [comment]
thank you again for your response ...😁
